src/wifi-setup/LICENSE

```diff
@@ -0,0 +1,373 @@
+Mozilla Public License Version 2.0
```

Comment:
Could we just remove this as it's covered by the top level LICENSE file with the same license?